recipes/flock/meta.yaml

```diff
@@ -0,0 +1,13 @@
+package:
+  name: flock
+  version: "1.0"
+source:
+  git_url: https://github.com/cristhomas/immport-test.git
```

Comment:
This one does not work? https://github.com/cristhomas/immport-test/archive/12b153db8d1e0192c449296d9348d2f33d279293.tar.gz